apps/emqx/test/emqx_persistent_session_SUITE.erl

```diff
@@ -131,7 +131,9 @@ init_per_group(Group, Config) when Group == ws; Group == ws_snabbkaffe ->
 init_per_group(Group, Config) when Group == tcp; Group == tcp_snabbkaffe ->
     [{port, 1883}, {conn_fun, connect} | Config];
 init_per_group(Group, Config) when Group == quic; Group == quic_snabbkaffe ->
-    [{port, 14567}, {conn_fun, quic_connect} | Config];
+    UdpPort = 1883,
```

Comment:
Q: should this be 1884?

Reply:
It is udp port number not tcp,

http/3 use the same udp port number as tcp port number . So we just follow that rule 😄